### ISSUE TITLE
Broken A/W/Alias pattern

### ISSUE THREAD

**Issue by GCuser99** [fafalone/WinDevLib#28]
I think the DeclareWide version of these should be aliased...
```vba
Public Declare PtrSafe Function StopTraceA Lib "advapi32" (ByVal TraceHandle As LongLong, ByVal InstanceName As String, Properties As EVENT_TRACE_PROPERTIES) As Long
Public Declare PtrSafe Function StopTraceW Lib "advapi32" Alias "StopTraceW" (ByVal TraceHandle As LongLong, ByVal InstanceName As LongPtr, Properties As EVENT_TRACE_PROPERTIES) As Long
Public DeclareWide PtrSafe Function StopTrace Lib "advapi32" (ByVal TraceHandle As LongLong, ByVal InstanceName As String, Properties As EVENT_TRACE_PROPERTIES) As Long
Public Declare PtrSafe Function QueryTraceA Lib "advapi32" (ByVal TraceHandle As LongLong, ByVal InstanceName As String, Properties As EVENT_TRACE_PROPERTIES) As Long
Public Declare PtrSafe Function QueryTraceW Lib "advapi32" Alias "QueryTraceW" (ByVal TraceHandle As LongLong, ByVal InstanceName As LongPtr, Properties As EVENT_TRACE_PROPERTIES) As Long
Public DeclareWide PtrSafe Function QueryTrace Lib "advapi32" (ByVal TraceHandle As LongLong, ByVal InstanceName As String, Properties As EVENT_TRACE_PROPERTIES) As Long
```

**Comment by fafalone** [comment]
Yup looks like I accidentally inserted the alias on the wrong line. Thanks.

**Comment by fafalone** [comment]
Fixed in v8.0.402, now available. Thanks for reporting!
